src/App.tsx

```diff
@@ -1,14 +1,28 @@
 import React from 'react';
-
 import './App.scss';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { PostList } from './components/PostList/PostList';
+import { PreparedPost } from './Types/Types';
+
+const preparedPosts: PreparedPost[] = posts.map((post) => {
+  const correctUser = users.find((user) => (user.id === post.userId));
+
+  const correctComments = comments.filter((comment) => (comment.postId === post.id));
+
+  const preparedPost = {
+    ...post,
+    user: correctUser || null,
+    comments: correctComments || null,
```

Comment:
could `null` ever be set to comments? if there are no comments, it will be just an empty array